### DISCUSSION TITLE
Does Floccus work Ok on Chromium?

### DISCUSSION THREAD

**Discussion by johnaaronrose** [floccusaddon/floccus#1776] [Q&A] [answered]
Android's Chromium does not allow Extensions to be downloaded from the Chrome Web Store. So I would have to use it from this website. Does Floccus work Ok in Chromium?

**Comment by marcelklehr** [comment, marked as answer]
Yes, should work fine.

**Reply by johnaaronrose** [reply]
I'll try it.

**Reply by johnaaronrose** [reply]
I've just installed it. I guess that this is more a Chromium question. I don't see any way in Chromium to 'run' it by changing its DEtails as in the Chrome downloaded version of Floccus running in Chromium. Any ideas?
PS I hate Google!

**Reply by marcelklehr** [reply]
You'll need to use the crx file

**Reply by johnaaronrose** [reply]
Please tell me where to find the crx file.

**Reply by marcelklehr** [reply]
https://github.com/floccusaddon/floccus/releases/tag/v5.3.4

**Reply by johnaaronrose** [reply]
What do I do with the crx file?

**Reply by marcelklehr** [reply]
1. Launch the Chrome browser, click the Chrome menu icon, Chrome menu icon, consisting of three vertical dots, then select More tools, and then select Extensions
2. Ensure that the Developer mode check box in the upper right corner is enabled.
3. Use drag-and-drop to drag the CRX file, and drop it into the Extensions window to install.

**Reply by johnaaronrose** [reply]
I was having a bad day yesterday and misunderstood how floccus works on Android. I think that the Floccus app (on Android)  synchronises with Ubuntu's Chromium browser's extension. On Android, each web site has to be input from the Floccus app manually into the Chromium browser's Mobile bookmarks folder. It would be nice if Android's Chromium app was able to Import (& merge) the bookmarks from Android's Floccus app. But that needs Chromium to allow the importing of a bookmarks file as Firefox etc do. I've attached the Ubuntu Chromium & the Android Floccus syncing screenshots.
![Ubuntu Chromium Screenshot](https://github.com/user-attachments/assets/362dd4d6-c74a-4e08-98cd-69d00f3837d3)
![Android Floccus Screenshot](https://github.com/user-attachments/assets/8b4e0f59-c649-4149-87a1-d1b0340f06d1)

**Reply by marcelklehr** [reply]
> On Android, each web site has to be input from the Floccus app manually into the Chromium browser's Mobile bookmarks folder. It would be nice if Android's Chromium app was able to Import (& merge) the bookmarks from Android's Floccus app. 

You are right. Floccus on Android (and on iOS as well) does not sync with any of the browser apps, due to technical limitations. You can, however, try the kiwi browser on android, which allows installing the floccus browser extension to sync bookmarks natively.

**Reply by johnaaronrose** [reply]
I've checked out the Kiwi browser already. But its website does not have a Ubuntu (or other Liinux desktop distro) version. Similar problem with Yandex as Chromium: it has some extensions installed *says that it is in Beta) but no mechanism yet for installing more: I'm reluctant to use as the Blog on its website is in Russian and it has Sync of bookmarks but it wants phone number

**Reply by johnaaronrose** [reply]
Alos, Ubuntu version of Yandex browser has an unsatisfied dependency!
I'll try and get Chromium to allow extensions from a library they create and control.

**Reply by marcelklehr** [reply]
> But its website does not have a Ubuntu (or other Liinux desktop distro) version. 

Why would you need an Ubuntu version of Kiwi Browser?

**Reply by johnaaronrose** [reply]
Yes, so I'm trying out the Kiwi browser on Android. It allows me to select a crx file to install an extension. So i tried the floccus crx file. However, it stated something about permissions and then gave  a 400 error. Developer's screenshot below.
![Android Floccus Screenshot](https://github.com/user-attachments/assets/89eaa757-5dbf-42a5-8528-5274c929d4c1)

**Reply by marcelklehr** [reply]
Ah, that's an open issue with Google Drive integration in floccus. For now you should be able to import a working profile after exporting it from your desktop browser

**Reply by johnaaronrose** [reply]
Interesting. In Chromium, it says that I'm not signed in, but I don't see any way to sign in! Also, editing my profile only allows me to change my avatar & theme. Nor do I see any way to export my profile from Chromium!

**Reply by marcelklehr** [reply]
I meant the floccus profile, not the chromium profile ;)

**Reply by johnaaronrose** [reply]
I've exported my profile from floccus extension in Ubuntu Chromium Ok: it's a .json file. Android's Kiwi browser only allows import of extension file types .zip, .crx, .user & .js. and thus won't accept the .json file.

**Reply by marcelklehr** [reply]
Ah, but you need to import it into floccus in kiwi, not kiwi. Install floccus extension in kiwi and then import the floccus profile into floccus.

**Reply by johnaaronrose** [reply]
I finally got Kiwi working as I wanted. Thanks for all your help.

**Reply by marcelklehr** [reply]
You're welcome :) Happy syncing!

**Comment by johnaaronrose** [comment]
I'm now trying to install floccus on another computer to sync with the existing one on the same profile. I'm getting a strange message after I dragged the crx file into the Extensions window of Chromium and took the Keep option when the response was that  the extension was unsafe and I tried to sign into my Google email a/c. It's an "Error 400:direct_uri_mismatch" also saying "access blocked: this app's request is invalid. Any ideas?

**Reply by marcelklehr** [reply]
Hey! Could you open a new issue for this?
